### PR TITLE
Fixed Bug-10101

### PR DESCRIPTION
**Pull request by divang26Sep**
Implementation/Solution details.

### CODE REVIEW

File: F1.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
 FEATURE 11
 FEATURE 11.0
 FEATURE 11.1
+FIX 11.0.1
```
Comment (github06022021) [inline F1.java]: It has a performance issue

Comment (divang26Sep) [reply F1.java]: Resolve perf issue.

**Review by github06022021 — APPROVED** [review]

**Comment by divang26Sep** [comment]
Added the resolution of Bug 10101

**Comment by github06022021** [comment]
Merging to master branch.

**Pull request merged**